roles/post_install_config/vars/main.yml

```diff
@@ -33,3 +33,11 @@ pulp_galaxy_permissions:
   - galaxy.change_synclist
   - galaxy.view_synclist
   - galaxy.add_synclist
+default_galaxy_importer_settings:
```

Comment:
@chouseknecht I put it there, but I'll happily remove writing any default settings. Problem with having default settings in this role is:

  1. It will easily get out of sync with galaxy-importer when PR gets merged (ie. https://github.com/ansible/galaxy-importer/pull/94) - needing role update, when not really needed.

  2. Given those are the defaults value within `galaxy-importer` there is no need to actually specify them. So idea would be to just write the behaviour we want to differ from defaults.

If you agree with only writing the diff, I'll update this PR accordingly